Packages/Web/templates/html/text.html.flt

```diff
@@ -24,6 +24,6 @@
   "><%= run.string %></span><%--%>
```

Comment:
we probably want to do the same thing over here, in the case where we have an attributed string